certsubmitter/certsubmitter.go

```diff
@@ -124,6 +124,12 @@ func checkSCT(sct *ct.SignedCertificateTimestamp, chain []*x509.Certificate, sv
 		errs = append(errs, &errors.SignatureVerificationError{Err: err})
 	}
 
+	// Verify SCT timestamp.
+	sctTimestamp := ct.TimestampToTime(sct.Timestamp)
+	if receivedAt.Sub(sctTimestamp).Seconds() < 0 {
```

Comment:
Use After() instead?

Reply:
My bad. Current monitor forgives gaps of up to 1 second. Let's stick to it for proper functionality alignment.